handlers/vm/lxc/lxc_test.go

```diff
@@ -16,7 +16,7 @@ func TestResourceName(t *testing.T) {
 
 func TestTypes(t *testing.T) {
 	assert := assert.New(t)
-	assert.Implements((*handlers.ResourceHandler)(nil), &LxcHandler{})
+	// assert.Implements((*handlers.InstanceResourceHandler)(nil), &LxcHandler{})
```

Comment:
Why is this commented?

Reply:
A little time ago,  InstanceResourceHandler was registered by each Handler. But now InstanceResourceHandler is not used by any handlers, because each Scheduler registering instanceSchedulerHandler to global variable of scheduler/schedule.go instead of InstanceResourceHandler. So each Hander need not implement InstanceResourceHandler. I deleted this comment.